package.json

```diff
@@ -19,14 +22,18 @@
   "author": "Vandebron <digital@vandebron.nl>",
   "license": "ISC",
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": ">= 2",
-    "@typescript-eslint/parser": "^2.1.0",
-    "eslint-config-prettier": "^6.2.0",
+    "@typescript-eslint/eslint-plugin": "^4.0.0",
+    "@typescript-eslint/parser": "^4.0.0",
+    "eslint": "^6.5.1",
+    "eslint-config-prettier": "^6.4.0",
     "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-prettier": ">= 3.1",
-    "eslint-plugin-react-hooks": "^2.2.0",
+    "eslint-plugin-prettier": "^3.3.0",
     "eslint-plugin-react": "^7.14.3",
-    "eslint": "^6.3.0",
-    "prettier": "1.18.2"
+    "eslint-plugin-react-hooks": "^2.2.0",
+    "prettier": "^2.2.1",
+    "typescript": "^4.1.3"
+  },
+  "devDependencies": {
+    "install-peers-cli": "^2.2.0"
```

Comment:
Curious about this, why do we need to have the peerdeps installed here? AFAIK you usually only need them when you have a runtime (e.g. tests) in the package itself, but we don't have that here right? 🤔

Reply:
Ah! This was added because linting would work at a high level fine (e.g. by pointing VS Code at the local package. However, `yarn lint` was failing because the dependencies weren't installed, which is why this is here now.

Reply:
Indeed it doesn't seem to work as I thought it would. I assumed that when ESLint in your project (i.e. not this library) interprets this config, it'd try to resolve the dependencies used here from the **project's** node_modules. But seems it's still trying to find them in the directory where this library lives. Quite annoying I must say 😄  But that answers both this and my first comment, so all good 👍